packages/schemas/src/types/jwt-customizer.ts

```diff
@@ -32,22 +32,31 @@ export const jwtCustomizerUserContextGuard = userInfoGuard.extend({
 
 export type JwtCustomizerUserContext = z.infer<typeof jwtCustomizerUserContextGuard>;
 
+export enum LogtoJwtTokenPath {
```

Comment:
this name confuses me - why is this called "path"

Reply:
This is used as part of API.

Reply:
LOG-8489, discussed offline, rename this to be `XyzType` to avoid naming misleading.